internal/plugin/grpc_provider.go

```diff
@@ -27,6 +25,16 @@ import (
 
 var logger = logging.HCLogger()
 
+// Some providers may generate quite large schemas, and the internal default
+// grpc response size limit is 4MB. 64MB should cover most any use case, and
+// if we get providers nearing that we may want to consider a finer-grained
+// API to fetch individual resource schemas.
+// Note: this option is marked as EXPERIMENTAL in the grpc API. We keep
+// this for compatibility, but recent providers all set the max message
+// size much higher on the server side, which is the supported method for
+// determining payload size.
```

Comment:
I think I already said this on a different version of this code somewhere else 🙃, but in case not:

"This option" in this comment is referring to `grpc.MaxRecvMsgSizeCallOption`, which was mentioned close to where this comment originally appeared but is no longer related to this factored-out constant alone. I'd suggest moving this second part of the comment back nearby the mention of that symbol so it'll hopefully be clearer to future maintainers what this is talking about.